roles/ood_user_reg_cloud/tasks/main.yml

```diff
@@ -47,10 +47,8 @@
 
 - name: Copy Celery service template
   template:
-    src: "{{ item }}"
-    dest: "/etc/systemd/system/{{ item }}"
-  with_items:
-    - celery.service
+    src: celery.j2
```

Comment:
We could still use the name celery.service.j2 as the source file.  The .service extension is descriptive since it's specific to systemd.  I agree we should have the final extension of .j2 for templates.  That helps make it clear that these files can't be used directly.  They must be processed by the template filter.

Reply:
resolved in 55e7243